internal/server/singleprocess/poll_project.go

```diff
@@ -59,6 +78,9 @@ func (pp *projectPoll) PollJob(
 			// any time queued per project.
 			SingletonId: fmt.Sprintf("poll/%s", p.Name),
 
+			// Project polling requires a data source to be configured for the project
+			DataSource: p.DataSource,
```

Comment:
I thought that jobs default to their project-configured data source at some point?

This probably does no harm, but was curious if you saw this break for some reason? They should default to it because the CLI generates jobs without data sources attached.

Reply:
I believe this was left over from when I had mistakenly moved the datasource job validation. But I think this shouldn't be harmful since it gets attached again in service_job later when we queue this job struct and validate a data source was configured with the project.

I can remove it back to how it was though if that's preferred 🤔